specs/protocol/configurability.md

```diff
@@ -0,0 +1,71 @@
+# OP Stack Configurability
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Protocol Parameters](#protocol-parameters)
+- [Admin Roles](#admin-roles)
+- [Service Roles](#service-roles)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+When deploying the OP Stack software to a production environment,
+certain parameters about the protocol can be configured. These
+configurations can include a variety of parameters which affect the
+resulting properties of the blockspace in question.
+
+There are three categories of OP Stack configuration options:
+
+- **Protocol Parameters**: Parameters and properties of the chain that may
+  be set at genesis and fixed for the lifetime of the chain, or may be
+  changeable through a privileged account.
+- **Admin Roles**: These roles can upgrade contracts, change role owners,
+  or update protocol parameters. These are typically cold/multisig wallets not
+  used directly in day to day operations.
+- **Service Roles**: These roles are used to manage the day-to-day
+  operations of the chain, and therefore are often hot wallets.
+
+## Protocol Parameters
+
+| Config Property                       | Description                                                                                                                  |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| Batch Inbox address                   | L1 address where calldata/blobs are posted (see [Batcher Transaction](../glossary.md#batcher-transaction)).                         |
+| Batch submission frequency            | Frequency with which batches are submitted to L1 (see [Batcher Transaction](../glossary.md#batcher-transaction)).            |
+| Chain ID                              | Unique ID of Chain used for TX signature validation.                                                                         |
+| Challenge Period                      | Length of time for which an output root can be removed, and for which it is not considered finalized.                                                                                                                                                             |
+| Compression ratio                     | How much compression the batch submitter applies to batches before submission (see [Channel](../glossary.md#channel)).       |
```

Comment:
This is not configurable, but rather a metric of performance of the batcher efficiency. Can you split out metrics in its own table? Or remove this property?